lua/entities/ent_jack_gmod_ezsolargenerator.lua

```diff
@@ -0,0 +1,247 @@
+AddCSLuaFile()
+
+ENT.Type = "anim"
+ENT.PrintName = "EZ Solar Panel"
+ENT.Author = "Jackarunda, AdventureBoots"
+ENT.Category = "JMod - EZ Misc."
+ENT.Information = ""
+ENT.Spawnable = true
+ENT.Base = "ent_jack_gmod_ezmachine_base"
+--
+ENT.MaxDurability = 50
+ENT.JModPreferredCarryAngles = Angle(90, 0, 0)
+ENT.MaxPower = 100
+ENT.SkyModifiers = {"clouds_", "_clouds", "cloudy_", "_cloudy", "stormy_", "storm_", "_storm"}
+--
+ENT.StaticPerfSpecs={
+	MaxDurability=50
+}
+ENT.DynamicPerfSpecs={
+	ChargeSpeed=1
+}
+function ENT:SetupDataTables()
+	self:NetworkVar("Int",0,"State")
+	self:NetworkVar("Int",1,"Grade")
+	self:NetworkVar("Float",0,"Progress")
+    self:NetworkVar("Float",1,"Visibility")
```

Comment:
screwy indent

Reply:
also, there's no need to network visibility
since all it affects is the charge rate

rather, we should network the entire calculated charge rate
visibility*mapNight*mapCloudy

that way, we can later display the networked value on the solar panel's hologram UI
for ease of use by the player

Reply:
*Facepalm* I'll fix it.